iam/terraform/account-specific/main/regional-log-subscription-filters/create-missing-log-groups.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Log groups subscribed to by Terraform are deployed per environment,
+# and may not exist at the time of running the account-specific Terraform
+# step. This script will create any missing log groups, so they can be
+# subscribed to.
```

Comment:
I got tired of doing this by hand & added a script here — it’s next to where Terraform will throw an error about missing log groups, so hopefully it’s naturally discoverable.